api/api.pb.txt

```diff
@@ -698,7 +730,7 @@ file {
     }
     field {
       name: "php_generic_services"
-      number: 19
+      number: 42
```

Comment:
I'm unfamiliar with this change.

I used `golang:1.9.3` docker image with `protoc-3.5.0-linux-x86_64.zip`, and did `make setup` and `make protos`.

Reply:
@stevvooe is this supposed to happen?

Reply:
Possibly the PR was opened before the Bump to 3.5? See https://github.com/docker/swarmkit/pull/2451

Reply:
Ah, looks like they were re-generated after that (likely with an outdated version of protoc; https://github.com/docker/swarmkit/commit/58e42ec9f7f3d241966b1d0f7bc34457888f6aa4

Reply:
Opened https://github.com/docker/swarmkit/pull/2503 to have just the proto changes